src/pages/patientView/patientHeader/PatientHeader.tsx

```diff
@@ -40,15 +34,15 @@ export default class PatientHeader extends React.Component<IPatientHeaderProps,
         );
     }
 
-    private getPopoverPatient(patient: typeof _ClinicalInformationData.patient) {
+    private getPopoverPatient(patient: ClinicalInformationData['patient']) {
         return patient && (
```

Comment:
As I said above, TypeScript settings include strictNullChecks=true and the type of ClinicalInformationData['patient'] includes undefined. This means TypeScript displays an error if you do not check for undefined.